### PR TITLE
update attack-surface-reduction-exploit-guard.md

### PR DESCRIPTION
**Pull request by MaratMussabekov**
issue #3278 
clarify license requirements

### CODE REVIEW

File: windows/security/threat-protection/windows-defender-exploit-guard/attack-surface-reduction-exploit-guard.md [hunk 1]
```diff
@@ -22,7 +22,7 @@ ms.date: 04/02/2019
...
+To use attack surface reduction rules, you need a Windows 10 Enterprise license. If you have Windows E5 license, it gives you the advanced management capabilities to power them. These include monitoring, analytics, and workflows available in [Windows Defender Advanced Threat Protection](../windows-defender-atp/windows-defender-advanced-threat-protection.md), as well as reporting and configuration capabilities in the M365 Security Center. These advanced capabilities aren't available with an E3 license or with Windows 10 Enterprise without subsciption, but you can use attack surface reduction rule events in Event Viewer to help facilitate deployment.
```
Comment (illfated) [inline windows/security/threat-protection/windows-defender-exploit-guard/attack-surface-reduction-exploit-guard.md]: minor adjustment (If you have **a** Windows E5 license)
```suggestion
To use attack surface reduction rules, you need a Windows 10 Enterprise license. If you have a Windows E5 license, it gives you the advanced management capabilities to power them. These include monitoring, analytics, and workflows available in [Windows Defender Advanced Threat Protection](../windows-defender-atp/windows-defender-advanced-threat-protection.md), as well as reporting and configuration capabilities in the M365 Security Center. These advanced capabilities aren't available with an E3 license or with Windows 10 Enterprise without subsciption, but you can use attack surface reduction rule events in Event Viewer to help facilitate deployment.
```

**Comment by MaratMussabekov** [comment]
Dear @illfated ,
you recommendations were applied, thank you!

**Comment by mypil** [comment]
@andreabichsel - Can you please assist to check, approve and merge the content for this PR based on issue #3278?

Thank you.

**Comment by mypil** [comment]
@andreabichsel @Justinha  - Can you please assist to check, approve and merge the content for this PR based on issue #3278?

Thank you.

**Comment by mypil** [comment]
@Justinha - We would like to follow up. Can you please assist to check, approve and merge the content for this PR based on issue #3278?

Thank you.

**Comment by mypil** [comment]
@Justinha - We would like to follow up. Do you have any updates on this? Thank you.

**Pull request merged**